models/Discord/Channels/DiscordCommandBugReport.js

```diff
@@ -55,7 +55,14 @@ class DiscordCommandBugReport extends DiscordChannel {
     }
 
     if(!body || !title) {
-      message.reply("No title/body detected in message. If this is a bug yell at someone!")
+      let title_or_body = !body ? "body" : "title"
+      title_or_body = (!body && !title) ? "body and title" : title_or_body
```

Comment:
```suggestion
      const title_or_body = (!body && !title) ? 
                              "body and title" : 
                                 !body ? "body" : "title"
```

Reply:
Oh that’s cool didn’t know you could do that